pages/common/install.md

```diff
@@ -23,3 +23,7 @@
 - Copy files and apply access/modification times of source to the destination:
 
 `install --preserve-timestamps {{path/to/source}} {{path/to/destination}}`
+
+- Copy files and create the directories at the destination if they don't exist:
+
+`install -D {{path/to/source}} {{path/to/destination}}`
```

Comment:
```suggestion
`install -D {{path/to/source_file1 path/to/source_file2 ...}} {{path/to/destination_directory}}`
```

Please fix any other code samples to be consistent with this one.

Reply:
Done